packages/flame/lib/src/components/isometric_tile_map_component.dart

```diff
@@ -108,6 +108,14 @@ class IsometricTileMapComponent extends PositionComponent {
     return Block(px, py);
   }
 
+  void setBlockValue(Block pos, int block) {
+    matrix[pos.y][pos.x] = block;
+  }
+
+  int getBlockValue(Block pos) {
```

Comment:
https://dart.dev/guides/language/effective-dart/design#avoid-starting-a-method-name-with-get

> AVOID starting a method name with get.
> 
> In most cases, the method should be a getter with get removed from the name. For example, instead of a method named getBreakfastOrder(), define a getter named breakfastOrder.
> 
> Even if the member does need to be a method because it takes arguments or otherwise isn’t a good fit for a getter, you should still avoid get. Like the previous guidelines state, either:
> 
>     Simply drop get and use a noun phrase name like breakfastOrder() if the caller mostly cares about the value the method returns.
> 
>     Use a verb phrase name if the caller cares about the work being done, but pick a verb that more precisely describes the work than get, like create, download, fetch, calculate, request, aggregate, etc.
>

Reply:
setters are ok? (I did a quick read on that doc and it seems so, lmk if I got it wrong)

Reply:
Yeah, I think they are if there is no other verb that describes it better, so I think it should be fine here.